packages/back-end/src/api/openapi/schemas/ExperimentResultsRow.yaml

```diff
@@ -0,0 +1,157 @@
+type: object
+required:
+  - experimentName
+  - experimentId
+  - snapshotId
+  - dimensionId
+  - dimensionName
+  - dimensionValue
+  - metricName
+  - metricId
+  - baselineVariationId
+  - variationName
+  - variationId
+  - variationUsers
+  - variationNumerator
+  - variationDenominator
+  - variationMean
+  - variationStdDev
+  - differenceType
+  - effect
+  - chanceToBeatControl
+  - relativeRisk
+  - pValue
+  - pValueAdjusted
```

Comment:
In the product, we use `adjustment` to refer to CUPED and `correction` to refer to multiple-comparisons.  So should this be `pValueCorrected` instead?

Speaking of which, how does this work for CUPED, Sequential, and Informative Priors?  Are we just including the final p-value / CTW / CIs?  Is there any way to tell what type of results are going to be returned?

I'm thinking about someone dumping all experiment results into a single SQL table in order to do a meta-analysis.  The rows should either be standardized and comparable, or have additional columns that allow for filtering (`statsEngine`, `regressionAdjustment`, `pValueConnectionMethod`, etc.)

Reply:
>So should this be pValueCorrected instead?

Yes, we could, although the language is a bit weird. I think it's better on average though.

>Speaking of which, how does this work for CUPED, Sequential, and Informative Priors? Are we just including the final p-value / CTW / CIs?

Just the final values. I was debating exporting the snapshot settings along with the results but thought that was too much data in one row. We could also tell users to pull this data from the snapshot collection but I think that's actually not exposed right now and we could handle that for them.